openedx/core/djangoapps/profile_images/views.py

```diff
@@ -124,15 +138,16 @@ class ProfileImageRemoveView(APIView):
 
     **Response for POST**
 
-        Users are authorized to delete their own profile images, while staff can delete images for
-        any account. All other users will receive a 404 to avoid leaking the existence of the account.
+        Requesting users who do not have staff access and try to remove another
+        user's profile image receive a 404 error.
 
-        This method will also return a 404 if no user exists with username "username".
+        If no user matches the "username" parameter, a 404 error is returned.
```

Comment:
Same questions as above re: "is returned" (question applies to the following two instances as well).